swarm/api/routes/wisdom.py

```
@@ -821,6 +853,12 @@ async def apply_wisdom_patches(
         404: Run or wisdom artifacts not found.
         400: Invalid patch_type or policy.
     """
+    try:
+        if run_id is not None:
+            validate_path_component(run_id, "run_id")
+    except ValueError as e:
+        raise HTTPException(status_code=400, detail={"error": "invalid_path", "message": str(e)})
```

Ruff (0.15.5)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)